app/controllers/UsersController.scala

```diff
@@ -37,11 +37,14 @@ case class ManageUserInfo(email: String,
                           id: String,
```

Comment:
Code changes to this file are not properly reformatted, please reformat all changes.